src/transaction/transaction.cpp

```diff
@@ -25,6 +27,21 @@ void getJsonValueIfNotNull(wrapper::json::OrderedJson const &json, std::string c
         val = std::make_shared<T>(tmpVal);
     }
 }
+
+std::string getTxSerializedHashIfRequired(Transaction const &tx)
```

Comment:
maybe rename to getTxMessageToSign ?

Reply:
Changed its name to `getSerializedTxMsg`. If you consider this name doesn't reflect its content, we could change it